### PR TITLE
Upgrade scalajs-react

### PR DESCRIPTION
**Pull request by cquiroz**
Big PR updating scalajs-react and dropping SemanticUI Toasts and utest

### CODE REVIEW

File: facade/src/test/scala/react/semanticui/collections/form/FormDropdownTests.scala [hunk 1]
```diff
@@ -1,32 +1,32 @@
 package react.semanticui.collections.form
```
Comment (toddburnside) [inline facade/src/test/scala/react/semanticui/collections/form/FormDropdownTests.scala:1]: Should this file be renamed?

File: facade/src/test/scala/react/semanticui/elements/button/ButtonGroupTests.scala [hunk 2]
```diff
@@ -1,38 +1,38 @@
 package react.semanticui.elements.button
```
Comment (toddburnside) [inline facade/src/test/scala/react/semanticui/elements/button/ButtonGroupTests.scala:1]: Rename file?

File: facade/src/test/scala/react/semanticui/elements/button/ButtonOrTests.scala [hunk 3]
```diff
@@ -1,16 +1,14 @@
 package react.semanticui.elements.button
```
Comment (toddburnside) [inline facade/src/test/scala/react/semanticui/elements/button/ButtonOrTests.scala:1]: Rename file?

File: facade/src/test/scala/react/semanticui/elements/button/ButtonTests.scala [hunk 4]
```diff
@@ -1,6 +1,5 @@
 package react.semanticui.elements.button
```
Comment (toddburnside) [inline facade/src/test/scala/react/semanticui/elements/button/ButtonTests.scala:1]: Rename file?

File: facade/src/test/scala/react/semanticui/elements/flag/FlagTests.scala [hunk 5]
```diff
@@ -1,24 +1,22 @@
 package react.semanticui.elements.flag
```
Comment (toddburnside) [inline facade/src/test/scala/react/semanticui/elements/flag/FlagTests.scala:1]: Rename file?

File: facade/src/test/scala/react/semanticui/modules/checkbox/CheckboxTests.scala [hunk 6]
```diff
@@ -1,29 +1,29 @@
 package react.semanticui.modules.checkbox
```
Comment (toddburnside) [inline facade/src/test/scala/react/semanticui/modules/checkbox/CheckboxTests.scala:1]: Rename file?

File: facade/src/test/scala/react/semanticui/modules/dropdown/DropdownMenuTests.scala [hunk 7]
```diff
@@ -1,28 +1,28 @@
 package react.semanticui.modules.dropdown
```
Comment (toddburnside) [inline facade/src/test/scala/react/semanticui/modules/dropdown/DropdownMenuTests.scala:1]: Rename file?

**Review by toddburnside — COMMENTED** [review]
Wow. What a huge, mind-numbing task! Thanks for taking this on.

File: facade/src/test/scala/react/semanticui/collections/form/FormDropdownTests.scala (hunk 1 above)
Comment (cquiroz) [reply facade/src/test/scala/react/semanticui/collections/form/FormDropdownTests.scala:1]: Yes I probably missed a few

File: facade/src/test/scala/react/semanticui/addons/confirm/ConfirmSuite.scala [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+// import japgolly.scalajs.react.vdom.html_<^._
+
+class ConfirmSuite extends munit.FunSuite {
+  test("render") {
+    //Modal cannot be tested without anmiationFrame
+    // val pusher = Confirm(content = "Abc")
+    // ReactTestUtils.withNewBodyElement { mountNode =>
+    //   pusher.renderIntoDOM(mountNode)
+    //   assert(mountNode.outerHTML == """<div><div class="header">Abc</div></div>""")
+    // }
```
Comment (rpiaggio) [inline facade/src/test/scala/react/semanticui/addons/confirm/ConfirmSuite.scala:13]: Did you leave this for later?

Comment (cquiroz) [reply facade/src/test/scala/react/semanticui/addons/confirm/ConfirmSuite.scala:13]: Eventually. testing this requires that we import an polyfill for `animationFrame`. I'll leave it as an exercise for the reader

**Pull request merged**